app/models/wiki.rb

```diff
@@ -101,7 +96,7 @@ def self.default_wiki
   end
 
   def self.get_or_create(language:, project:)
-    language = nil if project == 'wikidata'
+    language = nil if MULTILINGUAL_PROJECTS.include?(project)
```

Comment:
Rolling back to this means the single-language version of Wikisource won't work; they'll all get saved as `nil` even if they are supposed to be 'en.wikisource.org' or the like.